setup/requirements-fastapi.txt

```diff
@@ -4,3 +4,6 @@ typing==3.7.4
 uvicorn==0.11.2
 pyarango==1.3.3
 pre-commit==1.20.0
+fuzzysearch==0.7.0
+torch==1.4.0
+https://buddhanexus.kc-tbts.uni-hamburg.de/packages/buddhanexus_lang_analyzer-0.1-py3-none-any.whl
```

Comment:
does this work?

Reply:
it does! but it requires me to keep track that the package file on the BN-server is in good shape. Once I am done with the paper on this I am happy to let this point to a github-repository instead.